src/components/elements/favorites-list.tsx

```diff
@@ -15,8 +16,11 @@ const ShareButtons = () => {
   const urlParams = `/courses/favorites?favs=${favs.map(fav => `${fav.uuid}`).join(",")}`
   const copyUrl = isClient ? window.location.origin + urlParams : urlParams
 
+  const notify = () => toast("Copied!")
+
   const copyToClip = (text: string) => {
     copy(text).catch(_e => console.warn("An error occurred when copying to clipboard"))
+    notify()
```

Comment:
You don't need to define a separate function `notify`. Inside `copyToClip` you can just call `toast("Copied!")` directly.